sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/implementation/RadiologyInsightsClientImpl.java

```diff
@@ -2690,7 +2690,7 @@ public SyncPoller<BinaryData, BinaryData> beginInferRadiologyInsights(String id,
      * @return the {@link PollerFlux} for polling of response for the Radiology Insights request.
      */
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<RadiologyInsightsData, RadiologyInsightsInferenceResult>
+    public PollerFlux<RadiologyInsightsJob, RadiologyInsightsInferenceResult>
```

Comment:
This change is caused by typespec-azure-core upgrade and bug fix. It looks valid to me.